bHY6yh-de2985/cod-bxxqqq_bHY6yh-de2985.xml

```diff
@@ -0,0 +1,195 @@
+              <lb ed="#B"/>transibunt ait ihesus: </quote>
+            </p>
+            <p xml:id="bHY6yh-de2985-d1e251">
+               ¶ <quote xml:id="bHY6yh-de2985-Qd1e255">Idem dicit 
+              <lb ed="#B"/>lucas in euangelo capitulum 21 </quote>
+            </p>
+            <p xml:id="bHY6yh-de2985-d1e259">
+              <lb ed="#B"/>
+                <ref xml:id="bHY6yh-de2985-Rd1e273">Lucas in euangelo capitulo 10</ref> <quote xml:id="bHY6yh-de2985-Qd1e263"> <!--line jumps up-->Ihesus in<!--line jumps down-->trauit in quod<lb ed="#B" break="no"/>am
+                castellum et mulier quadam mar
+                <lb ed="#B"/>tha nomine excepit illum in domum 
+                <lb ed="#B"/>suam et huic erat soror nomine- 
+                <lb ed="#B"/>maria que etiam sedens secus 
+                <lb ed="#B"/>pedes domini audiebat verbum illius 
+                <lb ed="#B"/>martha autem satagebat circa 
+                <lb ed="#B"/>frequens ministerium que stetit et \
+                <lb ed="#B"/>ait domine non est tibi <unclear>cuquod?</unclear> soror. reliquit me
+                <lb ed="#B"/>solam ministrare dic ergo ut me adiu<lb ed="#B" break="no"/>uet
+                Et dixit illi <unclear>ibus</unclear>: Martha 
+                <lb ed="#B"/>martha sollicita es et turbaris erga 
+                <lb ed="#B"/>plurima porro vnum est necessarium. Maria 
+                <lb ed="#B"/>optimam partem elegit que non auferetur ab 
+                <lb ed="#B"/>ea </quote>
+            </p>
+            <p xml:id="bHY6yh-de2985-d1e290"> /
+              ¶ <ref xml:id="bHY6yh-de2985-Rd1e304">capitulo 11</ref> <quote xml:id="bHY6yh-de2985-Qd1e292">beati qui audiut verbum 
+              <lb ed="#B"/>dei et custodiunt illud</quote>
+            </p>
+            <p xml:id="bHY6yh-de2985-d1e298">
+              ¶ <quote xml:id="bHY6yh-de2985-Qd1e302">/ Et ait ue 
+              <lb ed="#B"/>uobis legislatoribus quia honeratis 
+              <lb ed="#B"/>homines honoribus que portari non 
+              <lb ed="#B"/>poaeient et ipsi vero digito vsro non 
+              <lb ed="#B"/>tangitis sarcinas. </quote>
+            </p>
+            <p xml:id="bHY6yh-de2985-d1e310">
+            <lb ed="#B"/>
+                <ref xml:id="bHY6yh-de2985-Rd1e328">Ioannes in euangeelo <!--line jumps up--> capitulo 8</ref> <quote xml:id="bHY6yh-de2985-Qd1e314">Qui ex <!--line jumps down--> deo est verba dei 
+                <lb ed="#B"/>audit. </quote>
+            </p>
+            <p xml:id="bHY6yh-de2985-d1e321"> /
+        ¶ <quote xml:id="bHY6yh-de2985-Qd1e323"> Ait ihesus: Si quis sermonem
+        <lb ed="#B"/>meum seruauerit mortem non gusta<lb ed="#B" break="no"/>bit 
+        in eternum. </quote>
+            </p>
+            <p xml:id="bHY6yh-de2985-d1e329">
+              ¶ <ref xml:id="bHY6yh-de2985-Rd1e349">/capitulo 10</ref> <quote xml:id="bHY6yh-de2985-Qd1e333">Ait ihesus si 
```

Comment:
same here. Is there a pilcrow and a slash. 

If so I would do 

```
¶ / <ref xml:id="bHY6yh-de2985-Rd1e349">...
```

Or dbcheck to make sure both pilcrow and slash are present.